### PR TITLE
Feature add transifex

### PR DESCRIPTION
**Pull request by rafaelje**
This PR add Transifex workflow:

## on Develop
- install python
- configure transifex
- pull locales before build with at least more than 80 percent complete
- push locales before build

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[main]
+host = https://www.transifex.com
+minimum_perc = 80
+lang_map = pt_BR: pt-rBR, ru_RU: ru-rRU, fr_FR: fr-rFR, es_MX: es-rMX, es_ES: es-rES, en_GB: en-rGB, ko_KR: ko-rKR
```
Comment (ajsb85) [inline .tx/config]: Is it mandatory?

File: ci/install.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@
 # ------------------------------------------------------------------------------
 
 # Catch if last commit come from last travis build to prevent loop
-if [[ $TRAVIS_COMMIT_MESSAGE != *"**beta**"* && $TRAVIS_COMMIT_MESSAGE != *"**version**"* && $TRAVIS_COMMIT_MESSAGE != *"**CHANGELOG.md**"* ]]; then
+if [[ $TRAVIS_COMMIT_MESSAGE != *"chore(release):"* && $TRAVIS_COMMIT_MESSAGE != *"**beta**"* && $TRAVIS_COMMIT_MESSAGE != *"**version**"* && $TRAVIS_COMMIT_MESSAGE != *"**CHANGELOG.md**"* ]]; then
```
Comment (ajsb85) [inline ci/install.sh]: Reeplace `chore` to `build`

File: .tx/config (hunk 1 above)
Comment (rafaelje) [reply .tx/config]: The only thing that is not mandatory is "minimum_perc" can be lowest or highest number but need to be more than 1% at least

Comment (ajsb85) [reply .tx/config]: My question is related to the `lang_map` because we add new translations often

Comment (rafaelje) [reply .tx/config]: Yes we need to keep the lang_map we are checking (iOS and Android) if we can make this more flexible with a regular expression

File: ci/install.sh (hunk 2 above)
Comment (rafaelje) [reply ci/install.sh]: This is the default commit generated with "conventional-github-releaser" I'm going to check if can be custom or make a --amend to git

**Pull request merged**